tests/testsuite/alt_registry.rs

```diff
@@ -289,6 +289,9 @@ fn cannot_publish_to_crates_io_with_registry_dependency() {
         )
         .build();
 
+    // Login so that we have the token available
```

Comment:
This test failed at an earlier point without access to the fake registry's token.